locales/en.json

```diff
@@ -285,10 +285,66 @@
   },
   "sidebar": {
     "analysis": {
+        "data": {
+            "highway": {
```

Comment:
why max_speed is not present here?

Reply:
See line [347](https://github.com/nrenner/brouter-web/pull/830/files/50e85509ac9c11bfc5b7c2542931048eb9bcb0ea#diff-c9218bed23f0aeb93d7e1a7d5f6e1345fae20f4b8692b27fb97fd0f37b3d26d7R347)